packages/vuetify/src/components/VExpansionPanel/_variables.scss

```diff
@@ -6,8 +6,10 @@ $expansion-panel-header-font-size: 0.9375rem !default;
 $expansion-panel-header-min-height: 48px !default;
 $expansion-panel-active-header-min-height: 64px !default;
 $expansion-panel-header-padding: 16px 24px !default;
-$expansion-panel-content-padding: 0 24px 16px !default;
+$expansion-panel-content-padding: 8px 24px 16px !default;
```

Comment:
are there any rtl considerations for this?

Reply:
It's `top inline bottom` so no